tasks/configure_option_merge.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: "Read {{ item.key }}"
+  become: yes
+  become_user: www-data
+  shell: "wp option get {{ item.key }} --path=/var/www/wordpress --format=json"
+  changed_when: False
+  register: result
+
+- name: Read output form json to yaml
```

Comment:
did you check whether yml output is possible in the `wp` command?

Reply:
yes its not possible